### PR TITLE
Generative FriendliAI Module

### PR DESCRIPTION
**Pull request by snowman39**
### What's being changed:

This PR adds FriendliAI generative module.

It follows the same manner as other generative-X modules.
I have tested this module on my local machine to confirm it works.

### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

**Comment by snowman39** [comment]
May I ask your review please? @antas-marcin

**Comment by antas-marcin** [comment]
@snowman39 in order for us to be able to merge your PR we need you to accept our CLA. Simply creating a comment with "I agree with CLA" will let us to merge your PR.

File: usecases/config/config_handler.go [hunk 1]
```diff
@@ -295,7 +295,7 @@ type CORS struct {
 const (
 	DefaultCORSAllowOrigin  = "*"
 	DefaultCORSAllowMethods = "*"
-	DefaultCORSAllowHeaders = "Content-Type, Authorization, Batch, X-Openai-Api-Key, X-Openai-Organization, X-Openai-Baseurl, X-Anyscale-Baseurl, X-Anyscale-Api-Key, X-Cohere-Api-Key, X-Cohere-Baseurl, X-Huggingface-Api-Key, X-Azure-Api-Key, X-Google-Api-Key, X-Google-Vertex-Api-Key, X-Google-Studio-Api-Key, X-Palm-Api-Key, X-Jinaai-Api-Key, X-Aws-Access-Key, X-Aws-Secret-Key, X-Voyageai-Baseurl, X-Voyageai-Api-Key, X-Mistral-Baseurl, X-Mistral-Api-Key, X-OctoAI-Api-Key, X-Anthropic-Baseurl, X-Anthropic-Api-Key "
+	DefaultCORSAllowHeaders = "Content-Type, Authorization, Batch, X-Openai-Api-Key, X-Openai-Organization, X-Openai-Baseurl, X-Anyscale-Baseurl, X-Anyscale-Api-Key, X-Cohere-Api-Key, X-Cohere-Baseurl, X-Huggingface-Api-Key, X-Azure-Api-Key, X-Google-Api-Key, X-Google-Vertex-Api-Key, X-Google-Studio-Api-Key, X-Palm-Api-Key, X-Jinaai-Api-Key, X-Aws-Access-Key, X-Aws-Secret-Key, X-Voyageai-Baseurl, X-Voyageai-Api-Key, X-Mistral-Baseurl, X-Mistral-Api-Key, X-OctoAI-Api-Key, X-Anthropic-Baseurl, X-Anthropic-Api-Key, X-FriendliAI-Api-Key, X-FriendliAI-Baseurl "
```
Comment (antas-marcin) [inline usecases/config/config_handler.go:298]: I don't know if this format will work in this case but just to be sure that all will be working can you change here the capitalisation of header names to this:
```
X-Friendliai-Api-Key, X-Friendliai-Baseurl
```

Comment (snowman39) [reply usecases/config/config_handler.go]: Thanks for your review :))

**Comment by snowman39** [comment]
I agree with CLA

**Review by antas-marcin — APPROVED** [review]

**Pull request merged**